docs/architecture/architecture-overview.adoc

```diff
@@ -0,0 +1,260 @@
+
+** *Step 1.1*: Install / Configure an OpenShift Ansible Broker (OAB)
++
+The Mobile Services are represented as APBs (Ansible Playbooks). For these playbooks to be read in, and represented through the Service Catalog, an https://docs.openshift.com/container-platform/3.7/architecture/service_catalog/ansible_service_broker.html#service-catalog-spec-file[Ansible Service Broker] must be created.
++
+image:images/5.x-Architecture-MC-ASB.png["My screenshot",width=1280, caption="Figure 1: Mobile Core Installation - Integrating Mobile Services into Service Catalog", title="Mobile Core Installation - Integrating Mobile Services into Service Catalog", link="images/5.x-Architecture-MC-ASB.png"]
++
+When the MC is installed, it will register a Broker with the Service Catalog. In some cases an OAB may already exist. A service broker is a server that conforms to the https://github.com/openservicebrokerapi/servicebroker/blob/v2.13/spec.md[OSB API] specification and manages a set of one or more services. The software could be hosted within your own OpenShift Container Platform cluster or elsewhere. The MC will create a specific type of Broker, an Ansible Service Broker. Once the ASB is created, it will try and invoke a GET on the its configured endpoint, to retrieve the catalog of services it has access to. The set of services are defined through Ansible playbooks, which for Mobile are located in https://github.com/aerogearcatalog[AeroGear Docker Hub]. These services are returned to the Serice Catalog and Service Class resources are created to represent each returned service type. At that point a user can see a representation of the services inside the Service Catalog.
+
+* *Step 2*: Extend the OpenShift UI, to offer a Mobile perspective
+
+The installation of MC also incorporates, extending the OpenShift UI. It does this by updating the Openshift master-config.yml. The MC UI is an angular application and is using angularjs version 1.5. Additional information on the customization of the Web UI is located https://docs.openshift.com/container-platform/3.7/install_config/web_console_customization.html[here].
+
+* *Step 3*: Create a Custom Resource Definition for Mobile App REpresentations
+
+Kubernetes offers the capability to define your own object kinds, using the https://kubernetes.io/docs/concepts/api-extension/custom-resources/[Customer Resource Definition] (CRD) concept. Rather than letting the Kubernetes core continiously expand and potentially get too unwieldly, CRDs provide a simple, yet flexible way to define your own object kinds and extend the Kubernetes core.
+
+The MC as part of its startup procedure creates a CRD for representing Mobile Clients. At a later stage when a Mobile App representation is created, this CRD is instantiated.
+
+.Example of CRD for a Mobile App Representation
+====
```

Comment:
I think it would be better to link to the actual CRD in the repo https://github.com/aerogear/mobile-cli/blob/master/artifacts/mobileclient_crd.yaml as it is likely to change over time